app/grandchallenge/core/static/css/core.css

```diff
@@ -242,3 +242,29 @@ a[aria-expanded="false"] .fa-chevron-down {
 .htmx-request.htmx-indicator {
     display: inline-block;
 }
+
+/* datatables */
+.dt-layout-start,
+.dt-layout-end {
+    padding-top: 0.2rem;
+    padding-bottom: 0.2rem;
```

Comment:
It might be worth putting this (all) into base.scss, or even creating a separate scss file that is added to the compilation chain, and then using the Boostrap variables. You could then use `$spacer` and the various factors to get to the consistent value, see https://getbootstrap.com/docs/4.6/utilities/spacing/. What do you think?

Reply:
Had a bit of a fight with the compilation chain, gave up and just put it in a seperate file next to the base.scss but have it import in the base.scss.